plugins/org.jkiss.dbeaver.ext.cubrid.ui/plugin.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.2"?>
+
+<plugin>
+
+    <extension point="org.jkiss.dbeaver.dataSourceView">
+        <view
+            class="org.jkiss.dbeaver.ext.generic.views.GenericConnectionPage"
+            id="org.jkiss.dbeaver.ext.cubrid.dataSourceWizardView"
+            targetID="org.jkiss.dbeaver.ext.ui.newConnectionWizard"                
+            dataSource="cubrid"
+            label="%dialog.connection.header"
+            icon="icons/cubrid_icon.png">
+        </view>
+        <view
+	        class="org.jkiss.dbeaver.ext.generic.views.GenericConnectionPage"
+	        id="org.jkiss.dbeaver.ext.cubrid.dataSourceEditorView"
+	        targetID="org.jkiss.dbeaver.ext.ui.editConnectionDialog"
+	        dataSource="cubrid"
+	        label="%dialog.connection.header"
+	        icon="icons/cubrid_icon.png">
+        </view>
+    </extension>
+
+    <extension point="org.jkiss.dbeaver.databaseEditor">
+        <editor
+            id="cubrid.table.ddl.view"
+            class="org.jkiss.dbeaver.ui.editors.sql.SQLSourceViewer"
+            main="false"
+            label="DDL"
+            description="DDL"
+            position="additions_end"
+            icon="#sql_text"
+            type="folder">
+            <objectType name="org.jkiss.dbeaver.ext.cubrid.model.CubridTable"/>
+        </editor>

```

Comment:
Not sure we want to declare it for DDL viewer again, we can use the definition `generic.table.ddl.view` if condition `object.supportsDDL()` is true.

Reply:
The DDL still not showing when I don't declare it again even `object.supportsDDL()` return true. I think it's because of  `<objectType>`. Since I use `CubridTable` but generic use `GenericTable`.